apps/ledger-live-mobile/src/components/WebPlatformPlayer/WebViewV2.tsx

```diff
@@ -105,6 +106,14 @@ function useUiHook(): Partial<UiHook> {
           onClose: onCancel,
         });
       },
+      "storage.get": async ({ key, storeId }) => {
+        const storageKey = storeId ? `${storeId}-${key}` : key;
+        return (await deviceStorage.get(storageKey)) as string;
+      },
+      "storage.set": ({ key, value, storeId }) => {
+        const storageKey = storeId ? `${storeId}-${key}` : key;
+        deviceStorage.save(storageKey, value);
+      },
```

Comment:
storeId will always be defined here, in all cases. If the client didn't specify a custom storeId, the appId will be inferred instead